docs/operator-docs/granule-workflows.md

```diff
@@ -0,0 +1,87 @@
+---
+id: granule-workflows
+title: Granule Workflows
+hide_title: true
+---
+
+# Granule Workflows
+
+## Ingest Kinesis
+
+1. Generate AWS Long Term Access Key
+
+* For steps on creating an AWS Access Key go to [AWS Long Term Access Key](../operator-docs/aws-long-term-access-key).
+* Copy the AWS Long Term Access Key.
+
+2. Add a Collection
+
+* Open another tab in your browser and go to your Earthdata Git repository URL or where your collection data files are located.
+* Open the collection configuration .json file that you will copy to the Cumulus dashboard.
+* In the Cumulus dashboard, add a new collection with an attached rule.
+* Go to the `Collections` page and then click on `Add Collection`.
+
+  ![Screenshot of Collections Page](assets/cd_collections_page.png)
+
+  * Cut and paste the collection in the template between the `{}` lines.
+
+  ![Screenshot of a Collection template for adding a new collection](assets/cd_collection.png)
+
+  * Click `Save` to create the new collection.
+  
+* Go to the `Rules` page and then click on `Add Rule`.
+
+  ![Screenshot of Rules page](assets/cd_rules_page.png)
+
+  * Fill out the template form. If you need details on the fields and how to complete a rule form go to [Create Rule In Cumulus](../operator-docs/create-rule-in-cumulus)
+
+  ![Screenshot of a Rules template for adding a new rule](assets/cd_add_rule.png)
+
+  * Click `Save` to create the new rule.
+
+3. Bulk Ingest Granules
```

Comment:
This section isn't describing bulk ingest. This section is describing bulk operations on granules already ingested into the system. Granules would need to be ingested using rules.

Reply:
@jjmccoy Can you make changes to this please?

Reply:
There are some comments on this page.